lib/puppet/indirector/face.rb

```diff
@@ -1,8 +1,8 @@
 require 'puppet/face'
 
 class Puppet::Indirector::Face < Puppet::Face
-  option "--terminus TERMINUS" do
-    summary "The indirector terminus to use."
+  option "--terminus _" + _("TERMINUS") do
+    summary _("The indirector terminus to use.")
     description <<-EOT
```

Comment:
Can we externalize the description fields delineated by heredocs here and elsewhere in this PR?